x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -209,20 +229,22 @@ export const CaseViewActivity = ({
             selectedSeverity={caseData.severity}
             onSeverityChange={onUpdateSeverity}
           />
-          <UserList
-            dataTestSubj="case-view-user-list-reporter"
-            email={emailContent}
-            headline={i18n.REPORTER}
-            users={[caseData.createdBy]}
-            userProfiles={userProfiles}
-          />
-          {userActionsData?.participants ? (
+          {caseUsers?.reporter ? (
```

Comment:
Should we show the `Unknown` user if we don't have the reporter instead of hiding the list?

I suppose another option would be to fallback to the case data right?

Reply:
Thanks! I will fallback to the case data.